src/sentry/monitors/endpoints/base_monitor_details.py

```diff
@@ -112,32 +112,34 @@ def update_monitor(
         if "project" in result and result["project"].id != monitor.project_id:
             raise ParameterValidationError("existing monitors may not be moved between projects")
 
+        old_slug = monitor.slug
+        old_status = monitor.status
+        if params:
+            monitor.update(**params)
+            self.create_audit_entry(
+                request=request,
+                organization=project.organization,
+                target_object=monitor.id,
+                event=audit_log.get_event_id("MONITOR_EDIT"),
+                data=monitor.get_audit_log_data(),
```

Comment:
not really the purpose of this PR but for future i think it'd be nice to have more details in the audit logs since i was looking at them yesterday (iwhat is actually being updated, or at the very least the values of fields that could've been updated, ie. including status in `get_audit_log_data()`)